src/collectors/ceph/ceph.py

```diff
@@ -12,22 +12,32 @@
 
 """
 
-try:
-    import json
-except ImportError:
-    import simplejson as json
-
+import json  # No need for simplejson fallback b/c ceph py modules are >=2.6
 import glob
 import os
 import subprocess
-
+import re
 import diamond.collector
+import diamond.convertor
+from diamond.collector import str_to_bool
+
+
+# Metric name/path separator
+_PATH_SEP = "."
```

Comment:
Good to make this a global, but there are a number of collectors that just simply use '.'

We should either convert everything to use a common variable, or use '.' throughout